Makefile

```diff
@@ -418,9 +369,13 @@ package-darwin: package-darwin-manoto
 		echo "-> Skipped: Can not generate a package on a non-OSX host."; \
 	fi;
 
-binaries: docker genassets linux windows darwin
+binaries: docker-assets docker-linux docker-windows darwin
+
+packages: require-version require-secrets clean-desktop docker-assets docker-package-windows docker-package-linux package-darwin
```

Comment:
Few things here:

1. We don't need to clean everything, just the desktop related stuff, so I have a new rule `clean-desktop`
2. We don't need to independently build the binaries since the `package-*` targets take care of that
3. Note how we explicitly use docker for everything except package-darwin.  This gives consistent behavior with how this used to work, namely that everything except darwin builds in docker.